### ISSUE TITLE
cheaty@centurix: yaml instead of json-template

### ISSUE THREAD

**Issue by uocxp** [linuxmint/cinnamon-spices-applets#1619]
is it possible to have the cheats written in yaml instead of json files. YAML is much easier and clear to write and read.

**Comment by NikoKrause** [comment]
@Centurix

**Comment by perette** [comment]
@uocxp Consider writing your cheatsheet in YAML and then feeding it through Node's yaml2json converter.

**Comment by Centurix** [comment]
I agree, YAML is easier to edit than JSON. It'd be nice if there was a native YAML reader/writer for the cinnamon applet space. In one of my other applets (Homestead Manager) I made a quick-and-dirty YAML reader to read the config from another application, but it's not terribly robust and only supports the very basics of YAML (and badly). 

I did consider making either a small GUI to edit these, something in Python/TKinter, and then thought about having some kind of central repository of sheets where you can add/edit without having to add the sheet to this repo every time. Requires some hosting/building which leads to a Cinnamon applet dependent on a third party service.

I'll have a dig around for a YAML solution. I think it is better. Is anyone here aware of a good YAML parser that I could maybe use that would work well here?
